post-processor/vagrant/parallels.go

```diff
@@ -25,6 +25,8 @@ func (p *ParallelsProvider) Process(ui packersdk.Ui, artifact packersdk.Artifact
 	// Create the metadata
 	metadata = map[string]interface{}{"provider": "parallels"}
 
+	copied := 0
+
```

Comment:
nit: use var for zero value

```suggestion
	var copied int

```

Reply:
I agree this is equivalent but is there a reason to prefer `var` over explicit zero instantiation for primitive types?

Reply:
General conventions for zero value that helps tell the reader we are interested in the zero value of this type. If it were to be any other value then we would declare and assign using the short variable operator.